common/key_value_database/platform/lmdb/lmdb.cpp

```diff
@@ -77,7 +79,7 @@ error::Error LmdbTransaction::Remove(const string &key) {
 }
 
 KeyValueDatabaseLmdb::KeyValueDatabaseLmdb() :
-	env_(lmdb::env::create()),
+	env_(new lmdb::env(lmdb::env::create())),
```

Comment:
I'd prefer `make_unique()` if possible, but YMMV.

Reply:
Ah, I *can* use it here. I was thinking that it's not C++11, but this is a platform file. Fixing!